### ISSUE TITLE
open position

### ISSUE THREAD

**Issue by Pandena** [Chopper1337/CS-CFG#6]
Hello, are you still an active developer in cs? We would like to hire you to work on a p2p skin trading system for our platform

**Comment by Chopper1337** [comment]
Yes, I am willing to contribute to such a platform but first you must trade all your cases to me, otherwise I will dismiss your request as nothing more than a boring joke :-)
